src/views/Home.vue

```diff
@@ -135,7 +136,7 @@ export default defineComponent({
 
 <style scoped>
 
-figure {
+figure{
```

Comment:
usually we keep a space between the bracket and the selector. Why did you remove it here?

Reply:
Yes, I was using another tag here so due to that may be this happened. I am updating it sir

Reply:
Updated sir